lib/mongo/collection.rb

```diff
@@ -483,6 +483,8 @@ def parallel_scan(cursor_count, options = {})
     # @option options [ true, false ] :bypass_document_validation Whether or
     #   not to skip document level validation.
     # @option options [ Hash ] :collation The collation to use.
+    # @option options [ Array ] :array_filters A set of filters specifying to which array elements
+    #   an update should apply.
```

Comment:
Sorry again, I think we don't need this here, as array filters aren't relevant to a replace_one. We do, however need the document on the update_one method on line 533.